neuralprophet/forecaster.py

```diff
@@ -2848,17 +2852,73 @@ def conformal_predict(
         df_cal = self.predict(calibration_df)
         # get predictions for test dataframe
         df_test = self.predict(df, **kwargs)
+        df_qr = df_test.copy()
+
         # initiate Conformal instance
         c = Conformal(
             alpha=alpha,
             method=method,
             n_forecasts=self.n_forecasts,
             quantiles=self.config_train.quantiles,
         )
-        # call Conformal's predict to output test df with conformal prediction intervals
-        df_forecast = c.predict(df=df_test, df_cal=df_cal)
+
+        if not show_qr:
+            # call Conformal's predict to output test df with conformal prediction intervals
+            df_forecast = c.predict(df=df_test, df_cal=df_cal)
+            # plot one-sided prediction interval width with q
+        else:
+            df_forecast = c.predict(df=df_test, df_cal=df_cal)
+
+            # get quantile regression intervals
```

Comment:
All of these modifications for advanced users (whatever they may be) should be done within the `c.predict()`. `conformal_predict()` should be as lightweight as the forecaster.py file is already long enough. You can create an additional helper function in Conformal class for advanced users and have `c.predict()` call if it helps not make it too overloaded.